src/cmdstan-guide/generate_quantities_config.qmd

```diff
@@ -32,7 +86,70 @@ checks:
                   output file=bernoulli_ppc.csv
 ```
 
-The `fitted_params` file must be a Stan CSV file; attempts to use a regular CSV file
+The output file `bernoulli_ppc.csv` consists of just the values for the variables declared in the generated quantities block, i.e., `theta_rep` and the elements of `y_sim`:
+
+```
+# model = bernoulli_ppc_model
+# method = generate_quantities
+#   generate_quantities
+#     fitted_params = bernoulli_fit.csv
+# id = 0 (Default)
+# data
+#   file = bernoulli.data.json
+# init = 2 (Default)
+# random
+#   seed = 2135140492 (Default)
+# output
+#   file = bernoulli_ppc.csv
+#   diagnostic_file =  (Default)
+#   refresh = 100 (Default)
+theta_rep,y_sim.1,y_sim.2,y_sim.3,y_sim.4,y_sim.5,y_sim.6,y_sim.7,y_sim.8,y_sim.9,y_sim.10
+0.2,0,0,1,0,0,0,0,0,1,0
+0.3,1,0,0,1,0,1,0,0,0,0
+0.8,1,0,1,1,1,1,1,1,1,0
+0.1,0,0,0,0,0,1,0,0,0,0
+0.3,0,0,0,0,0,0,1,1,1,0
+```
+
+_Note_: the only relevant analysis of the resulting CSV output is computing per-column statistics; this can easily be done in Python, R, Excel or similar,
```

Comment:
I think we can just drop this paragraph.  

Also, try to keep lines down to 80 characters so we can look at diffs on GitHub.